src/lib/components/notes/stories/Note.stories.svelte

```diff
@@ -51,18 +56,34 @@
 </Template>
 
 <Story name="default" args={{ note: notes[0] }} />
+
 <Story name="page-level note" args={{ note: page_note }} />
+
 <Story name="editable" args={{ note: { ...notes[1], edit_access: true } }} />
+
 <Story
   name="private access"
   args={{ note: { ...notes[1], access: "private" } }}
 />
+
 <Story
   name="collaborators access"
   args={{ note: { ...notes[1], access: "organization" } }}
 />
+
 <Story name="note with HTML" args={{ note: { ...notes[2], content: html } }} />
+
 <Story
   name="render using PDF"
   args={{ pdf: writable(load(url)), note: { ...notes[2], content: html } }}
 />
+
+<Story name="Example: CJI">
```

Comment:
```suggestion
<Story name="Excerpt from rotated page">
```

Let's name this against the condition we're checking against.